### ISSUE TITLE
Prompts to change language 

### ISSUE THREAD

**Issue by roblourens** [microsoft/vscode#117727]
- Have some language packs installed that you aren't using
- Open a devcontainer for the first time

![image](https://user-images.githubusercontent.com/323878/109244049-da84d100-7792-11eb-8cd4-9055f96217ec.png)

Is there some special logic that installs language packs on a new remote? Other workspace extensions don't get installed automatically. Then I guess they show that notification whenever they are installed

**Comment by dbaeumer** [comment]
@sandy081 can you help me with this since I have not written the code. 

If I remember correctly we sync LP extensions to the remote which might trigger the `onDidInstallExtension` event here: https://github.com/microsoft/vscode/blob/main/src/vs/workbench/contrib/localizations/browser/localizations.contribution.ts#L53

To we have any means to find out that this is a extension sync and don't show the notification at all. We only sync the LP to ensure messages coming from the remote are correctly translated in case the user has selected a different locale.

**Comment by sandy081** [comment]
> If I remember correctly we sync LP extensions to the remote

I am now aware of any such sync to remote. May I know where do we do that?

**Comment by dbaeumer** [comment]
@sandy081 I really thought you implemented that. How would otherwise the LP come into the remote in dev containers? Without having them there we couldn't localize any strings in the extension host.

**Comment by dbaeumer** [comment]
I was able to reproduce this for a dev container under Docker. After creating a new container my local German LP got synced over. 

<img width="353" alt="capture" src="https://user-images.githubusercontent.com/1931590/109950807-45776000-7cdd-11eb-9c96-ef17d2e52798.png">

So someone must sync the stuff. @chrmarti do you know something about this

**Comment by chrmarti** [comment]
Yes, Remote-Containers installs all language packs remotely. Can I somehow filter for the currently active language?
https://github.com/microsoft/vscode-remote-containers/blob/10f0259d3a5b7794429e810c06215b05b7a14c29/src/extension/extensionUtils.ts#L340

**Comment by dbaeumer** [comment]
You can access the current language using `platform.language` (see https://github.com/microsoft/vscode/blob/main/src/vs/base/common/platform.ts#L165).

Syncing only that one would make sense and should address the issue since we are filtering against the active language in the did change event.
